pkg/cmd/repo/fork/fork.go

```diff
@@ -48,16 +50,19 @@ func NewCmdFork(f *cmdutil.Factory, runF func(*ForkOptions) error) *cobra.Comman
 	}
 
 	cmd := &cobra.Command{
-		Use:   "fork [<repository>]",
-		Args:  cobra.MaximumNArgs(1),
+		Use:   "fork [<repository>] [-- <gitflags>...]",
+		Args:  cobra.MaximumNArgs(2),
 		Short: "Create a fork of a repository",
 		Long: `Create a fork of a repository.
 
-With no argument, creates a fork of the current repository. Otherwise, forks the specified repository.`,
+With no argument, creates a fork of the current repository. Otherwise, forks the specified repository.
+
+If '--clone' is specified, you can pass additional 'git clone' flags by listing them after '--'.`,
```

Comment:
"If the '--clone' flag is specified, additional 'git clone' flags can be passed in by listing them after '--'."

I think this wording is a bit more clear. 

cc @ampinsk

Reply:
If `--clone` wasn't specified, a prompt will be shown and the user will still have an opportunity to clone. Therefore, gitflags might take effect regardless of whether `--clone` was explicitly used or not.